autoreject/bayesopt.py

```diff
@@ -55,7 +55,9 @@ def bayes_opt(f, initial_x, acquisition, max_iter=100, debug=False,
     for i in range(max_iter):
         gp.fit(np.array(X)[:, None], np.array(y))
         new_x = scipy.optimize.fmin_l_bfgs_b(acquisition(gp, best_f),
-                                             x0=best_x, approx_grad=True)[0]
+                                             x0=best_x, approx_grad=True,
+                                             epsilon=1e-16,
```

Comment:
@dengemann this was the nasty bug that was sensitive to tiny numerical differences in the epochs !